src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.js

```diff
@@ -114,7 +115,11 @@ uiModules.get('apps/management')
     }
 
     // All system indices begin with a period.
-    return indices.filter(index => !index.name.startsWith('.'));
+    const filtered = indices.filter(index => !index.name.startsWith('.'));
+    if (filtered.length > MAX_NUMBER_OF_MATCHING_INDICES) {
+      return filtered.slice(0, MAX_NUMBER_OF_MATCHING_INDICES);
```

Comment:
What about removing the `if` block and always return the slice?

Reply:
I can do that!